data/templates/frr/daemons.frr.tmpl

```diff
@@ -14,44 +33,79 @@ babeld=yes
 sharpd=no
 pbrd=no
 bfdd=yes
-staticd=yes
```

Comment:
Why do you remove staticd?

Reply:
I dont, please read the comment at top of the daemons-file.

Original:

https://github.com/FRRouting/frr/blob/stable/9.0/tools/etc/frr/daemons

VyOS-edition:

https://github.com/vyos/vyos-1x/pull/2280/files#diff-4f2b1f67d87a8ad6d278fef1c3f91c573d8a2c339060d7e2712e3e431d3c88bbR2

Its commented out at top of the daemons list simply because watchfrr, zebra, mgmtd and staticd daemons are always started.

They are not even included in the original daemons-file from FRR 9.0/stable.